bconf/MCU_ESP32.h

```diff
@@ -43,6 +43,7 @@
 #define WIFI_MRF24					0
 #define WIFI_ESP8266				0
 #define WIFI_ESP32					1
+#define ETH_ESP32					0
```

Comment:
Had to define a specific ETH macro for ESP32 as ETH_W5x00 ones are very specific to arduino and would mean too many changes into a big part of the codebase.